### PR TITLE
Cleanup code

### PR DESCRIPTION
**Pull request by ho-stanley**
Code cleanup in database, controller and models.

### CODE REVIEW

File: app/Http/Controllers/ActorController.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ public function store(Request $request)
     {
         $this->authorize('create', Actor::class);
 
-        $validated = $request->validate([
+        $request->validate([
             'name' => 'bail|required|string|max:255',
```
Comment (jagestedt) [inline app/Http/Controllers/ActorController.php:53]: don't recognize this syntax, what does it do?

File: database/seeders/MovieSeeder.php [hunk 2]
```diff
@@ -18,36 +18,26 @@ class MovieSeeder extends Seeder
...
 
             if (isset($Movies['id']) && isset($Movies['title']) && isset($Movies['overview']) && isset($Movies['release_date']) && isset($Movies['poster_path']) && isset($Movies['backdrop_path']) && isset($Casts['cast'][5])) {
-
                 $director = '';
                 $producer = '';
                 $writer = '';
 
-
-                //Find Director, Writer, and Producer 
+                //Find Director, Writer, and Producer
```
Comment (jagestedt) [inline database/seeders/MovieSeeder.php:40]: add a space

**Review by jagestedt — APPROVED** [review]

File: app/Http/Controllers/ActorController.php (hunk 1 above)
Comment (ho-stanley) [reply app/Http/Controllers/ActorController.php:53]: I assume you mean on line 53? It's a regular array key value mapping. I define the valid input from the request.

**Review by jagestedt — CHANGES_REQUESTED** [review]
- You can explain the first comment (about the syntax) tonight. 
- And I found one comment in MovieSeeder without a space before the text

**Review by jagestedt — APPROVED** [review]

**Review by LinusUnghammar — APPROVED** [review]
Nice!

**Pull request merged**